### PR TITLE
Tidy up check location page - RIP-53

### PR DESCRIPTION
**Pull request by autotelik**
fix missing layout
add redirect url to config
tidy up locale
get feature specs passing

### CODE REVIEW

**Comment by autotelik** [comment]
:eyes:

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 class ApplicationController < FloodRiskEngine::ApplicationController
   include Pundit
 
+  layout ->(_) { "appplication" }
+
```
Comment (woodpigeon) [inline app/controllers/application_controller.rb]: Is `layout 'application'` simpler and a less confusing way to do it?

File: config/initializers/flood_risk_engine.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+FloodRiskEngine.configure do |config|
+  config.redirection_url_on_location_unchecked = "/pages/contact_ea_location"
+end
```
Comment (woodpigeon) [inline config/initializers/flood_risk_engine.rb]: I have a spearate PR #6 that also sets the default layout for the engine - can be merged later

File: app/controllers/application_controller.rb (hunk 1 above)
Comment (autotelik) [inline app/controllers/application_controller.rb]: I saw you adding some configuration to set the layout in other PR? so this was a place holder to pull that in ... i believe when it comes dynamically from a  variable you need this format but could be wrong ?

Comment (autotelik) [inline app/controllers/application_controller.rb]: e.g    layout ->(_) { DigitalServicesCore.layout }

File: spec/support/page_objects/check_location_page_object.rb [hunk 3]
```diff
@@ -1,18 +1,11 @@
 class CheckLocationPageObject < BasePageObject
```
Comment (woodpigeon) [inline spec/support/page_objects/check_location_page_object.rb]: @autotelik @reggieb I've got some reservations about using feature spec + page objects here and maybe we need a meeting about our testing approach in the two apps. I'm glad we are not using feature specs in the engine as specs run *sooo* much faster there. I thought we might test the page content in the engine using controller/vew specs to keep things fast. The page content and workflow in the journey is the responsibility of the engine (unless i18n-overridden in the app, which I don't think we need to do) so should be tested there. Tim S is writing cucumber scenarios which might well do the job of the feature specs in this PR. And we don't want to duplicate testing. So we need to think about what kind of tests we have in which component to keep tests running quickly, avoid duplication, and keep the code as simple as possible and as easy as possible to encourage our successors to write specs. Ug.

File: Gemfile [hunk 4]
```diff
@@ -1,9 +1,9 @@
 source "https://rubygems.org"
 ruby "2.2.3"
 
-gem "flood_risk_engine", git: "https://github.com/EnvironmentAgency/flood-risk-engine", branch: 'develop'
+gem "flood_risk_engine", git: "https://github.com/EnvironmentAgency/flood-risk-engine", branch: "develop"
 
-gem "copy_kit", git: 'https://github.com/EnvironmentAgency/copykit'
+gem "copy_kit", git: "https://github.com/EnvironmentAgency/copykit"
```
Comment (woodpigeon) [inline Gemfile]: Why do we include copykit? - thought running it was an external process

File: spec/support/page_objects/check_location_page_object.rb (hunk 3 above)
Comment (autotelik) [inline spec/support/page_objects/check_location_page_object.rb]: ok

File: Gemfile (hunk 4 above)
Comment (autotelik) [inline Gemfile]: I added FR specific configuration & task to generate forms/modules etc in right format for FR so it should be run from here now

Comment (autotelik) [inline Gemfile]: I'll delete it once views/forms stable

**Comment by autotelik** [comment]
:eyes:  Suggested changes to layout made

File: config/initializers/flood_risk_engine.rb (hunk 2 above)
Comment (autotelik) [inline config/initializers/flood_risk_engine.rb]: @woodpigeon agreed. removed from FO

**Pull request merged**